app/src/main/java/co/tinode/tindroid/media/VxCard.java

```diff
@@ -72,7 +73,7 @@ public int merge(Mergeable another) {
         int changed = super.merge(another);
         VxCard avc = (VxCard)another;
         if (avc.avatar != null) {
-            avatar = avc.avatar;
+            avatar = !Tinode.isNull(avc.avatar) ? avc.avatar : null;
```

Comment:
`Tinode.isNull(avc.avatar)` is always false

Reply:
Done.